spec/requests/api/v1/moves_controller_sorting_spec.rb

```diff
@@ -192,27 +197,22 @@
         end
 
         context 'when name' do
-          let(:people) { object_ids.map { |p_id| Person.find(p_id) } }
           let(:object_name) { 'person' }
-          let(:move_data) {
-            Move.all
-                                .sort_by { |move| move.profile.last_name }
-                                .map { |move| move.profile.person }
-          }
+          let(:last_names) { object_ids.map { |person_id| Person.find(person_id).latest_profile.last_name } }
```

Comment:
We could clean up this line, maybe something like:
`Person.find(object_ids).include(:profile).pluck(:last_name)`

Reply:
hmm.. not sure about that: `Person.find(object_ids)` won't guarantee the order and the test will sometimes fail